packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -59,87 +67,131 @@ class FramesBarPlotly {
     );
   }
 
+  // Return a list of all of traces in trace index order:
+  // e.g., [
+  //         GPU Good Trace Data,   // array index gpuGoodTraceIndex
+  //         GPU Jank Trace Data,   // array index gpuJankTraceIndex
+  //         GPU Select Trace Data, // array index gpuSelectTraceIndex
+  //         CPU Good Trace Data,   // array index cpuGoodTraceIndex
+  //         CPU Jank Trace Data,   // array index cpuJankTraceIndex
+  //         CPU Select Trace Data, // array index cpuSelectTraceIndex
+  //       ]
   static List<Data> createFPSTraces() {
+    final List<Data> allTraces = [];
+
     // Strange plotly bug with initial setup of x,y.  If x and y are empty array
     // then the first entry, for each trace, isn't rendered but hover does
     // display the Y value.  So prime each trace with some data.  Added
     // at x-axis coord of xCoordNotUsed (-1) (hide rangemode: nonnegative
     // displays at 0 and greater) and y is zero.
-    final Data traceCpuGood = Data(
-      y: [yCoordNotUsed],
-      x: [xCoordNotUsed],
-      type: 'bar',
-      legendgroup: 'good_group',
-      name: 'CPU',
-      hoverinfo: 'y+name',
-      marker: Marker(
-        color: colorToCss(mainCpuColor),
-      ),
-      width: [0],
-    );
 
-    final Data traceGpuGood = Data(
-      y: [yCoordNotUsed],
-      x: [xCoordNotUsed],
-      type: 'bar',
-      legendgroup: 'good_group',
-      name: 'GPU',
-      hoverinfo: 'y+name',
-      marker: Marker(
-        color: colorToCss(mainGpuColor),
-      ),
-      width: [0],
-    );
+    // trace GPU Good
+    allTraces.insert(
+        gpuGoodTraceIndex,
+        Data(
+          y: [yCoordNotUsed],
+          x: [xCoordNotUsed],
+          type: 'bar',
+          legendgroup: 'good_group',
+          name: 'GPU',
+          hoverinfo: 'y+name',
+          marker: Marker(
+            color: colorToCss(mainGpuColor),
+          ),
+          width: [0],
+        ));
```

Comment:
can you add the trailing comma between the final two parentheses? that will bring the final paren and the semicolon down to be vertically inline with `allTraces...`. If you add the trailing comma between the final two parens and then run dartfmt it should fix it automatically.

Same for all cases below.

Reply:
Yep already did that when you made the earlier suggestion.

Reply:
sorry I thought you had already pushed that commit. Nevermind :)